api-gateway/src/main/resources/static/scripts/bill-details/bill-details.template.html

```diff
@@ -12,7 +12,9 @@
         </div>
     </div>
     <br>
-    <br>
+    <div class="alert alert-warning" ng-if="$ctrl.bills.billStatus === 'OVERDUE'">
+        NOTICE: This bill is overdue. Please notify client to pay as soon as possible.
```

Comment:
I like your idea of concise and informative notification for overdue bills. 
Great job on implementing the overdue bill alert on the Bill Details page! 
This is a valuable addition that ensures timely action on overdue bills.